src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -1552,7 +1563,7 @@ module Util =
         let genArgs = transformGenArgs com ctx genArgs
         let path = makeFullNamePath ent.FullName genArgs
         let expr = mkStructExpr path fields // TODO: range
-        if isCopyableEntity com Set.empty ent
+        if isStruct || isCopyableEntity com Set.empty ent
```

Comment:
@alexswan10k Please remove change as `isCopyableEntity` already handles value types correctly.

Reply:
I don't think this is accurate. If you remove the value type check, structs are always rendered as Lrc. I accept the fix might be in the wrong place and need to be in isCopyableEntity, but I am not sure this logically follows. 

What has wrapped status got to do with if something is copyable or not? The logic is - value types should be unwrapped, surely?

Reply:
@alexswan10k 
> structs are always rendered as Lrc

We already have a lot of tests for struct classes, struct records, struct unions, etc., and they don't render as Lrc.
Do you have an example?

Reply:
Sure. Check out StructRecord in RecordTests and associated tests. It is wrapped in a Lrc before this change.

Reply:
@alexswan10k The REPL hasn't been updated yet with latest, but you're right. What isCopyableType/isCopyableEntity does is to check if we can derive from "Copy". In this case we cannot (since one of the fields is not Copy).

So your change is correct and I was wrong.

Reply:
No worries :)

I believe it just tells us if something implements the Clone trait or not. It is a pretty close approximation most of the time I agree, but both a ```Lrc<RawStruct>``` and a ```RawStruct``` are both cloneable so I am not sure this actually tells us the whole story.